### ISSUE TITLE
"wrong-type-argument stringp nil" when opening notes with helm-bibtex

### ISSUE THREAD

**Issue by eren-rudy** [org-roam/org-roam-bibtex#239]
**Describe the bug**
Can't open associated org-roam note when navigating citations with helm-bibtex.

**To Reproduce**
1. Sources imported to Zotero
2. Using Better Bibtex to export the file `zotero.bib` in BibLaTex format
3. Run helm-bibtex (using M-x : helm-bibtex)
4. Search for desired citation
5. Press TAB to bring up action options
6. Press F8 to open notes

**Expected behavior**
I expect the org-roam node associated with the citation to open. Alternatively, if there is no org-roam node, it should create one. I have tried both cases without success.

**Things I've tried**
- `doom doctor`seemed to be complaining that org-ref wasn't loading, so I added `(package! org-ref)` in my `packages.el`, which seemed to resolve that issue.
- This workflow worked perfectly when I first set up ORB. When I noticed this error today, I tried updating all of the dependent packages without success.
- I thought this might be an issue with the bibtex format, so I've tried with both BibTex and BibLaTex (as generated by Zotero), changing the corresponding bibtex-dialect variable, without success. I've also tried removing all citations from the .bib file, other than one that I know I have used succesfully in the past, also without success.
- Everything behaves as expected when I run 'orb-insert-link', allowing me to search my references indexed from the .bib file and create a new note (or insert a link for one that already exists).

**ORB configuration**
1. How the package is loaded (I am using Doom)
packages.el:
`(package! org-ref)`
`(package! org-roam-bibtex`
`  :recipe (:host github :repo "org-roam/org-roam-bibtex"))`
`(unpin! org-roam)`
`(unpin! bibtex-completion helm-bibtex ivy-bibtex)`

`(use-package! org-roam-bibtex`
`  :after org-roam`
`  :config`
`  (require 'org-ref))`

2. ORB options set to non-default values (e.g. 'orb-templates')

`(setq org-roam-bibtex-mode t)`
`(setq bibtex-completion-bibliography "~/Library/Mobile Documents/com~apple~CloudDocs/zotero.bib")`
`(setq bibtex-dialect 'biblatex)`
`(setq org-ref-cite-insert-version 2)`

**Environment (please complete the following information):**
 - ORB commit or MELPA package version: 20220117.801
 - Org Roam commit or MELPA package version: 20220126.554
 - Org Mode version: 20210929
 - Emacs framework [e.g. Doom, Spacemacs, Prelude, none]: Doom
 - Emacs version [e.g. 26.3, 27.1]: 27.2
 - OS: [e.g. Linux, macOS, Windows]: macOS 11.6

**Screenshots**
Here is the debug output generated via `toggle-debug-on-error`

```elisp
Debugger entered--Lisp error: (wrong-type-argument stringp nil)
  expand-file-name(nil)
  find-file-noselect(nil nil nil nil)
  find-file-other-window(nil)
  bibtex-completion-edit-notes-default(("newportDeepWorkRules2016"))
  bibtex-completion-edit-notes(("newportDeepWorkRules2016"))
  helm-bibtex-edit-notes("newportDeepWorkRules2016")
  helm-execute-selection-action-1()
  helm-execute-selection-action()
  helm-internal((((name . "BibTeX entries") (header-name . #f(compiled-function (name) #<bytecode 0x1ff3eab8f255>)) (candidates . helm-bibtex-candidates) (keymap keymap (f13 lambda nil (interactive) (helm-select-nth-action 12)) (f12 lambda nil (interactive) (helm-select-nth-action 11)) (f11 lambda nil (interactive) (helm-select-nth-action 10)) (f10 lambda nil (interactive) (helm-select-nth-action 9)) (f9 lambda nil (interactive) (helm-select-nth-action 8)) (f8 lambda nil (interactive) (helm-select-nth-action 7)) (f7 lambda nil (interactive) (helm-select-nth-action 6)) (f6 lambda nil (interactive) (helm-select-nth-action 5)) (f5 lambda nil (interactive) (helm-select-nth-action 4)) (f4 lambda nil (interactive) (helm-select-nth-action 3)) (f3 lambda nil (interactive) (helm-select-nth-action 2)) (f2 lambda nil (interactive) (helm-select-nth-action 1)) (menu-bar keymap (help-menu keymap ...)) (help keymap (109 . helm-help)) (23 . #f(compiled-function () ... #<bytecode 0x1ff4062b5e11>)) (f1 lambda nil (interactive) (helm-select-nth-action 0)) (8 keymap (109 . helm-help) (104 . undefined) (8 . undefined) (99 . helm-customize-group) (4 . helm-enable-or-switch-to-debug)) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap (57 . helm-execute-selection-action-at-nth-+9) (56 . helm-execute-selection-action-at-nth-+8) (55 . helm-execute-selection-action-at-nth-+7) (54 . helm-execute-selection-action-at-nth-+6) (53 . helm-execute-selection-action-at-nth-+5) (52 . helm-execute-selection-action-at-nth-+4) (51 . helm-execute-selection-action-at-nth-+3) (50 . helm-execute-selection-action-at-nth-+2) (49 . helm-execute-selection-action-at-nth-+1) (63 . helm-help) (110 . #f(compiled-function () ... #<bytecode 0x1ff4062b1901>)) (108 . helm-display-line-numbers-mode) (62 . helm-toggle-truncate-line) (21 . helm-refresh) (6 . helm-follow-mode) (9 . helm-insert-or-copy) (11 . helm-kill-selection-and-quit) (25 . helm-yank-selection) (37 . helm-exchange-minibuffer-and-header-line) (95 . helm-toggle-full-frame) (45 . helm-swap-windows)) (67108987 . helm-enlarge-window) (67108989 . helm-narrow-window) (19 . undefined) (24 keymap (57 . helm-execute-selection-action-at-nth-+9) (56 . helm-execute-selection-action-at-nth-+8) (55 . helm-execute-selection-action-at-nth-+7) (54 . helm-execute-selection-action-at-nth-+6) (53 . helm-execute-selection-action-at-nth-+5) (52 . helm-execute-selection-action-at-nth-+4) (51 . helm-execute-selection-action-at-nth-+3) (50 . helm-execute-selection-action-at-nth-+2) (49 . helm-execute-selection-action-at-nth-+1) (2 . helm-resume-list-buffers-after-quit) (98 . helm-resume-previous-session-after-quit) (6 . helm-quit-and-find-file)) (11 . helm-delete-minibuffer-contents) (67108896 . helm-toggle-visible-mark-forward) (0 . helm-toggle-visible-mark) (C-M-up . helm-scroll-other-window-down) (C-M-down . helm-scroll-other-window) (M-prior . helm-scroll-other-window-down) (M-next . helm-scroll-other-window) (12 . helm-recenter-top-bottom-other-window) (left . helm-previous-source) (right . helm-next-source) (15 . helm-next-source) ...) (action ("Open PDF, URL or DOI" . helm-bibtex-open-any) ("Open URL or DOI in browser" . helm-bibtex-open-url-or-doi) ("Insert citation" . helm-bibtex-insert-citation) ("Insert reference" . helm-bibtex-insert-reference) ("Insert BibTeX key" . helm-bibtex-insert-key) ("Insert BibTeX entry" . helm-bibtex-insert-bibtex) ("Attach PDF to email" . helm-bibtex-add-PDF-attachment) ("Edit notes" . helm-bibtex-edit-notes) ("Show entry" . helm-bibtex-show-entry) ("Add PDF to library" . helm-bibtex-add-pdf-to-library)) (requires-pattern . 0) (filtered-candidate-transformer helm-bibtex-candidates-formatter helm-fuzzy-highlight-matches) (match helm-mm-exact-match helm-mm-match) (redisplay . identity) (header-line . "C-j: Open PDF, URL or DOI (keeping ses...") (multimatch . t) (group . helm)) ((name . "Fallback options") (match (lambda (_candidate) t)) (candidates . bibtex-completion-fallback-candidates) (no-matchplugin) (nohighlight) (action lambda (candidate) (bibtex-completion-fallback-action candidate helm-pattern)))) nil nil nil #f(compiled-function () #<bytecode 0x1ff3eabd20a5>) "*helm bibtex*" nil nil nil)
  apply(helm-internal ((((name . "BibTeX entries") (header-name . #f(compiled-function (name) #<bytecode 0x1ff3eab8f255>)) (candidates . helm-bibtex-candidates) (keymap keymap (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff4062b5e11>)) (f1 lambda nil ... ...) (8 keymap ... ... ... ... ...) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ...) (67108987 . helm-enlarge-window) (67108989 . helm-narrow-window) (19 . undefined) (24 keymap ... ... ... ... ... ... ... ... ... ... ... ...) (11 . helm-delete-minibuffer-contents) (67108896 . helm-toggle-visible-mark-forward) (0 . helm-toggle-visible-mark) (C-M-up . helm-scroll-other-window-down) (C-M-down . helm-scroll-other-window) (M-prior . helm-scroll-other-window-down) (M-next . helm-scroll-other-window) (12 . helm-recenter-top-bottom-other-window) (left . helm-previous-source) (right . helm-next-source) (15 . helm-next-source) ...) (action ("Open PDF, URL or DOI" . helm-bibtex-open-any) ("Open URL or DOI in browser" . helm-bibtex-open-url-or-doi) ("Insert citation" . helm-bibtex-insert-citation) ("Insert reference" . helm-bibtex-insert-reference) ("Insert BibTeX key" . helm-bibtex-insert-key) ("Insert BibTeX entry" . helm-bibtex-insert-bibtex) ("Attach PDF to email" . helm-bibtex-add-PDF-attachment) ("Edit notes" . helm-bibtex-edit-notes) ("Show entry" . helm-bibtex-show-entry) ("Add PDF to library" . helm-bibtex-add-pdf-to-library)) (requires-pattern . 0) (filtered-candidate-transformer helm-bibtex-candidates-formatter helm-fuzzy-highlight-matches) (match helm-mm-exact-match helm-mm-match) (redisplay . identity) (header-line . "C-j: Open PDF, URL or DOI (keeping ses...") (multimatch . t) (group . helm)) ((name . "Fallback options") (match (lambda ... t)) (candidates . bibtex-completion-fallback-candidates) (no-matchplugin) (nohighlight) (action lambda (candidate) (bibtex-completion-fallback-action candidate helm-pattern)))) nil nil nil #f(compiled-function () #<bytecode 0x1ff3eabd20a5>) "*helm bibtex*" nil nil nil))
  helm((((name . "BibTeX entries") (header-name . #f(compiled-function (name) #<bytecode 0x1ff3eab8f255>)) (candidates . helm-bibtex-candidates) (keymap keymap (f13 lambda nil (interactive) (helm-select-nth-action 12)) (f12 lambda nil (interactive) (helm-select-nth-action 11)) (f11 lambda nil (interactive) (helm-select-nth-action 10)) (f10 lambda nil (interactive) (helm-select-nth-action 9)) (f9 lambda nil (interactive) (helm-select-nth-action 8)) (f8 lambda nil (interactive) (helm-select-nth-action 7)) (f7 lambda nil (interactive) (helm-select-nth-action 6)) (f6 lambda nil (interactive) (helm-select-nth-action 5)) (f5 lambda nil (interactive) (helm-select-nth-action 4)) (f4 lambda nil (interactive) (helm-select-nth-action 3)) (f3 lambda nil (interactive) (helm-select-nth-action 2)) (f2 lambda nil (interactive) (helm-select-nth-action 1)) (menu-bar keymap (help-menu keymap ...)) (help keymap (109 . helm-help)) (23 . #f(compiled-function () ... #<bytecode 0x1ff4062b5e11>)) (f1 lambda nil (interactive) (helm-select-nth-action 0)) (8 keymap (109 . helm-help) (104 . undefined) (8 . undefined) (99 . helm-customize-group) (4 . helm-enable-or-switch-to-debug)) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap (57 . helm-execute-selection-action-at-nth-+9) (56 . helm-execute-selection-action-at-nth-+8) (55 . helm-execute-selection-action-at-nth-+7) (54 . helm-execute-selection-action-at-nth-+6) (53 . helm-execute-selection-action-at-nth-+5) (52 . helm-execute-selection-action-at-nth-+4) (51 . helm-execute-selection-action-at-nth-+3) (50 . helm-execute-selection-action-at-nth-+2) (49 . helm-execute-selection-action-at-nth-+1) (63 . helm-help) (110 . #f(compiled-function () ... #<bytecode 0x1ff4062b1901>)) (108 . helm-display-line-numbers-mode) (62 . helm-toggle-truncate-line) (21 . helm-refresh) (6 . helm-follow-mode) (9 . helm-insert-or-copy) (11 . helm-kill-selection-and-quit) (25 . helm-yank-selection) (37 . helm-exchange-minibuffer-and-header-line) (95 . helm-toggle-full-frame) (45 . helm-swap-windows)) (67108987 . helm-enlarge-window) (67108989 . helm-narrow-window) (19 . undefined) (24 keymap (57 . helm-execute-selection-action-at-nth-+9) (56 . helm-execute-selection-action-at-nth-+8) (55 . helm-execute-selection-action-at-nth-+7) (54 . helm-execute-selection-action-at-nth-+6) (53 . helm-execute-selection-action-at-nth-+5) (52 . helm-execute-selection-action-at-nth-+4) (51 . helm-execute-selection-action-at-nth-+3) (50 . helm-execute-selection-action-at-nth-+2) (49 . helm-execute-selection-action-at-nth-+1) (2 . helm-resume-list-buffers-after-quit) (98 . helm-resume-previous-session-after-quit) (6 . helm-quit-and-find-file)) (11 . helm-delete-minibuffer-contents) (67108896 . helm-toggle-visible-mark-forward) (0 . helm-toggle-visible-mark) (C-M-up . helm-scroll-other-window-down) (C-M-down . helm-scroll-other-window) (M-prior . helm-scroll-other-window-down) (M-next . helm-scroll-other-window) (12 . helm-recenter-top-bottom-other-window) (left . helm-previous-source) (right . helm-next-source) (15 . helm-next-source) ...) (action ("Open PDF, URL or DOI" . helm-bibtex-open-any) ("Open URL or DOI in browser" . helm-bibtex-open-url-or-doi) ("Insert citation" . helm-bibtex-insert-citation) ("Insert reference" . helm-bibtex-insert-reference) ("Insert BibTeX key" . helm-bibtex-insert-key) ("Insert BibTeX entry" . helm-bibtex-insert-bibtex) ("Attach PDF to email" . helm-bibtex-add-PDF-attachment) ("Edit notes" . helm-bibtex-edit-notes) ("Show entry" . helm-bibtex-show-entry) ("Add PDF to library" . helm-bibtex-add-pdf-to-library)) (requires-pattern . 0) (filtered-candidate-transformer helm-bibtex-candidates-formatter helm-fuzzy-highlight-matches) (match helm-mm-exact-match helm-mm-match) (redisplay . identity) (header-line . "C-j: Open PDF, URL or DOI (keeping ses...") (multimatch . t) (group . helm)) ((name . "Fallback options") (match (lambda (_candidate) t)) (candidates . bibtex-completion-fallback-candidates) (no-matchplugin) (nohighlight) (action lambda (candidate) (bibtex-completion-fallback-action candidate helm-pattern)))) nil nil nil #f(compiled-function () #<bytecode 0x1ff3eabd20a5>) "*helm bibtex*" nil nil nil)
  apply(helm ((((name . "BibTeX entries") (header-name . #f(compiled-function (name) #<bytecode 0x1ff3eab8f255>)) (candidates . helm-bibtex-candidates) (keymap keymap (f13 lambda nil ... ...) (f12 lambda nil ... ...) (f11 lambda nil ... ...) (f10 lambda nil ... ...) (f9 lambda nil ... ...) (f8 lambda nil ... ...) (f7 lambda nil ... ...) (f6 lambda nil ... ...) (f5 lambda nil ... ...) (f4 lambda nil ... ...) (f3 lambda nil ... ...) (f2 lambda nil ... ...) (menu-bar keymap ...) (help keymap ...) (23 . #f(compiled-function () ... #<bytecode 0x1ff4062b5e11>)) (f1 lambda nil ... ...) (8 keymap ... ... ... ... ...) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ... ...) (67108987 . helm-enlarge-window) (67108989 . helm-narrow-window) (19 . undefined) (24 keymap ... ... ... ... ... ... ... ... ... ... ... ...) (11 . helm-delete-minibuffer-contents) (67108896 . helm-toggle-visible-mark-forward) (0 . helm-toggle-visible-mark) (C-M-up . helm-scroll-other-window-down) (C-M-down . helm-scroll-other-window) (M-prior . helm-scroll-other-window-down) (M-next . helm-scroll-other-window) (12 . helm-recenter-top-bottom-other-window) (left . helm-previous-source) (right . helm-next-source) (15 . helm-next-source) ...) (action ("Open PDF, URL or DOI" . helm-bibtex-open-any) ("Open URL or DOI in browser" . helm-bibtex-open-url-or-doi) ("Insert citation" . helm-bibtex-insert-citation) ("Insert reference" . helm-bibtex-insert-reference) ("Insert BibTeX key" . helm-bibtex-insert-key) ("Insert BibTeX entry" . helm-bibtex-insert-bibtex) ("Attach PDF to email" . helm-bibtex-add-PDF-attachment) ("Edit notes" . helm-bibtex-edit-notes) ("Show entry" . helm-bibtex-show-entry) ("Add PDF to library" . helm-bibtex-add-pdf-to-library)) (requires-pattern . 0) (filtered-candidate-transformer helm-bibtex-candidates-formatter helm-fuzzy-highlight-matches) (match helm-mm-exact-match helm-mm-match) (redisplay . identity) (header-line . "C-j: Open PDF, URL or DOI (keeping ses...") (multimatch . t) (group . helm)) ((name . "Fallback options") (match (lambda ... t)) (candidates . bibtex-completion-fallback-candidates) (no-matchplugin) (nohighlight) (action lambda (candidate) (bibtex-completion-fallback-action candidate helm-pattern)))) nil nil nil #f(compiled-function () #<bytecode 0x1ff3eabd20a5>) "*helm bibtex*" nil nil nil))
  helm(:sources (((name . "BibTeX entries") (header-name . #f(compiled-function (name) #<bytecode 0x1ff3eab8f255>)) (candidates . helm-bibtex-candidates) (keymap keymap (f13 lambda nil (interactive) (helm-select-nth-action 12)) (f12 lambda nil (interactive) (helm-select-nth-action 11)) (f11 lambda nil (interactive) (helm-select-nth-action 10)) (f10 lambda nil (interactive) (helm-select-nth-action 9)) (f9 lambda nil (interactive) (helm-select-nth-action 8)) (f8 lambda nil (interactive) (helm-select-nth-action 7)) (f7 lambda nil (interactive) (helm-select-nth-action 6)) (f6 lambda nil (interactive) (helm-select-nth-action 5)) (f5 lambda nil (interactive) (helm-select-nth-action 4)) (f4 lambda nil (interactive) (helm-select-nth-action 3)) (f3 lambda nil (interactive) (helm-select-nth-action 2)) (f2 lambda nil (interactive) (helm-select-nth-action 1)) (menu-bar keymap (help-menu keymap ...)) (help keymap (109 . helm-help)) (23 . #f(compiled-function () ... #<bytecode 0x1ff4062b5e11>)) (f1 lambda nil (interactive) (helm-select-nth-action 0)) (8 keymap (109 . helm-help) (104 . undefined) (8 . undefined) (99 . helm-customize-group) (4 . helm-enable-or-switch-to-debug)) (20 . helm-toggle-resplit-and-swap-windows) (C-tab . undefined) (67108897 . helm-toggle-suspend-update) (3 keymap (57 . helm-execute-selection-action-at-nth-+9) (56 . helm-execute-selection-action-at-nth-+8) (55 . helm-execute-selection-action-at-nth-+7) (54 . helm-execute-selection-action-at-nth-+6) (53 . helm-execute-selection-action-at-nth-+5) (52 . helm-execute-selection-action-at-nth-+4) (51 . helm-execute-selection-action-at-nth-+3) (50 . helm-execute-selection-action-at-nth-+2) (49 . helm-execute-selection-action-at-nth-+1) (63 . helm-help) (110 . #f(compiled-function () ... #<bytecode 0x1ff4062b1901>)) (108 . helm-display-line-numbers-mode) (62 . helm-toggle-truncate-line) (21 . helm-refresh) (6 . helm-follow-mode) (9 . helm-insert-or-copy) (11 . helm-kill-selection-and-quit) (25 . helm-yank-selection) (37 . helm-exchange-minibuffer-and-header-line) (95 . helm-toggle-full-frame) (45 . helm-swap-windows)) (67108987 . helm-enlarge-window) (67108989 . helm-narrow-window) (19 . undefined) (24 keymap (57 . helm-execute-selection-action-at-nth-+9) (56 . helm-execute-selection-action-at-nth-+8) (55 . helm-execute-selection-action-at-nth-+7) (54 . helm-execute-selection-action-at-nth-+6) (53 . helm-execute-selection-action-at-nth-+5) (52 . helm-execute-selection-action-at-nth-+4) (51 . helm-execute-selection-action-at-nth-+3) (50 . helm-execute-selection-action-at-nth-+2) (49 . helm-execute-selection-action-at-nth-+1) (2 . helm-resume-list-buffers-after-quit) (98 . helm-resume-previous-session-after-quit) (6 . helm-quit-and-find-file)) (11 . helm-delete-minibuffer-contents) (67108896 . helm-toggle-visible-mark-forward) (0 . helm-toggle-visible-mark) (C-M-up . helm-scroll-other-window-down) (C-M-down . helm-scroll-other-window) (M-prior . helm-scroll-other-window-down) (M-next . helm-scroll-other-window) (12 . helm-recenter-top-bottom-other-window) (left . helm-previous-source) (right . helm-next-source) (15 . helm-next-source) ...) (action ("Open PDF, URL or DOI" . helm-bibtex-open-any) ("Open URL or DOI in browser" . helm-bibtex-open-url-or-doi) ("Insert citation" . helm-bibtex-insert-citation) ("Insert reference" . helm-bibtex-insert-reference) ("Insert BibTeX key" . helm-bibtex-insert-key) ("Insert BibTeX entry" . helm-bibtex-insert-bibtex) ("Attach PDF to email" . helm-bibtex-add-PDF-attachment) ("Edit notes" . helm-bibtex-edit-notes) ("Show entry" . helm-bibtex-show-entry) ("Add PDF to library" . helm-bibtex-add-pdf-to-library)) (requires-pattern . 0) (filtered-candidate-transformer helm-bibtex-candidates-formatter helm-fuzzy-highlight-matches) (match helm-mm-exact-match helm-mm-match) (redisplay . identity) (header-line . "C-j: Open PDF, URL or DOI (keeping ses...") (multimatch . t) (group . helm)) ((name . "Fallback options") (match (lambda (_candidate) t)) (candidates . bibtex-completion-fallback-candidates) (no-matchplugin) (nohighlight) (action lambda (candidate) (bibtex-completion-fallback-action candidate helm-pattern)))) :full-frame t :buffer "*helm bibtex*" :input nil :preselect #f(compiled-function () #<bytecode 0x1ff3eabd20a5>) :candidate-number-limit 500 :bibtex-candidates (("2016 Newport, Cal Deep Work: Rules for..." ("date" . "2016") ("author" . "Newport, Cal") ("title" . "Deep Work: Rules for Focused Success i...") ("=type=" . "book") ("=key=" . "newportDeepWorkRules2016"))) :bibtex-local-bib nil)
  helm-bibtex(nil)
  funcall-interactively(helm-bibtex nil)
  call-interactively(helm-bibtex record nil)
  command-execute(helm-bibtex record)
  execute-extended-command(nil "helm-bibtex" #("helm" 0 4 (ws-butler-chg chg)))
  funcall-interactively(execute-extended-command nil "helm-bibtex" #("helm" 0 4 (ws-butler-chg chg)))
  call-interactively(execute-extended-command nil nil)
  command-execute(execute-extended-command)
```

**Comment by myshevchuk** [comment]
The backtrace indicates that ORB functions are not actually called when you try to create notes with `M-x helm-bibtex`, `F8`, `Open Notes`. This can happen when you do not have `org-roam-bibtex-mode` running. Calling `org-roam-bibtex-mode` arranges for a few tweaks, in particular, it makes Helm-bibtex use Org-roam for note taking.

Indeed, the following statement in your `config.el`: `(setq org-roam-bibtex-mode t)` does not do what you think it should do. From the docstring of the `org-roam-bibtex-mode` variable:
```
Setting this variable directly does not take effect;
either customize it (see the info node (emacs)Easy Customization)
or call the function org-roam-bibtex-mode.
```
This is in fact true for any major or minor Emacs mode. One should call the corresponding function instead of setting the mode variable. Mode variables are sort of internal - they are often used in subroutines to determine whether a particular mode is running or not, but setting them directly - setting any Emacs variable - does not normally execute code.

So in short, put `(org-roam-bibtex-mode +1)` somewhere in your config. You can wrap this call in an `after! org-roam` statement, so as to load ORB after Org-roam.

**Comment by eren-rudy** [comment]
That did the trick, thank you! Somehow I had tricked myself into thinking that setting the variable manually had done something, I guess it had just been a very long time since I had quit Emacs.
